scripts/formgen.ts

```diff
@@ -0,0 +1,142 @@
+import { PDFDocument, PDFField, PDFCheckBox } from 'pdf-lib'
+import { readFile } from 'fs/promises'
+import * as path from 'path'
+import { unzip } from '../src/util'
+
+const loadFile = async (path: string): Promise<PDFDocument> => {
+  const file = await readFile(path)
+  const bytearray = file.slice(0, file.byteLength)
+  return await PDFDocument.load(bytearray)
+}
+
+const normalizeName = (name: string): string =>
+  name.replaceAll(/[^a-zA-Z0-9]/g, '')
```

Comment:
Getting this error when running formGen
```
(node:15542) UnhandledPromiseRejectionWarning: TypeError: name.replaceAll is not a function
    at normalizeName (/mnt/c/Users/grims/github/UsTaxes/scripts/formgen.ts:13:8)
    at fieldFunction (/mnt/c/Users/grims/github/UsTaxes/scripts/formgen.ts:16:16)
    at Array.map (<anonymous>)
    at buildSource (/mnt/c/Users/grims/github/UsTaxes/scripts/formgen.ts:67:47)
    at /mnt/c/Users/grims/github/UsTaxes/scripts/formgen.ts:116:16
    at step (/mnt/c/Users/grims/github/UsTaxes/scripts/formgen.ts:52:23)
    at Object.next (/mnt/c/Users/grims/github/UsTaxes/scripts/formgen.ts:33:53
```
Might be because of lack of support? looks like a relatively new function (ES12)

Reply:
Interesting, yes it seems ts-node transpiles down and then your version of node doesn't have that function. `.replace` is fine here anyway.